README.md

```diff
@@ -426,6 +427,13 @@ If a user agent partitions service workers using this scheme, there is no cross-
 
 [Service workers are disabled](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Privacy/State_Partitioning) in Firefox when Dynamic Partitioning is enabled, but they are [working on implementing a partitioned service worker solution](https://bugzilla.mozilla.org/show_bug.cgi?id=1495241).
 
+## Security and Considerations
+
+### Security
+
+This proposal offers a small security improvement over third-party/cross-site cookies as currently available on the platform, because they require the `__Host-` prefix and `Secure` attribute.
```

Comment:
You should mention the mild DoS risk that https://github.com/WICG/CHIPS/blob/main/README.md#avoid-a-large-memory-footprint mitigates.

I don't think partitioning, as such, offers the security improvement. Rather, like with guidance to restrict new features to [secure contexts](https://w3c.github.io/webappsec-secure-contexts/#security-considerations), you're taking the opportunity of a new feature to push developers onto more-secure options.

Reply:
> You should mention the mild DoS risk that https://github.com/WICG/CHIPS/blob/main/README.md#avoid-a-large-memory-footprint mitigates.

Could you elaborate? Is the idea that state proliferation will cause browsers to take up too much memory on client devices?

> I don't think partitioning, as such, offers the security improvement.

One thing I didn't include about partitioning is that if a partitioned cookie is ever compromised then the user's identity is only known for a particular top-level site. Do you think that is worth mentioning here?

> ...like with guidance to restrict new features to secure contexts, you're taking the opportunity of a new feature to push developers onto more-secure options.

Ack'd, changed the language a bit to reflect this point.

Reply:
> Could you elaborate? Is the idea that state proliferation will cause browsers to take up too much memory on client devices?

Yeah, that partitioning cookies uses N times as much space/memory where N is the number of top-level sites the partitioned site is embedded within.

> One thing I didn't include about partitioning is that if a partitioned cookie is ever compromised then the user's identity is only known for a particular top-level site. Do you think that is worth mentioning here?

Yeah, it's plausible that a site would be vulnerable to XSS when embedded as an iframe, and partitioning the cookies makes that attack useless. The attacker would have to actually navigate to the site, which is much more visible to users, although not in a way that would prompt them to reset their cookies.

Reply:
Ok. I added a few sentences about each point here.